inheritance/src/main/java/inheritance/Restaurant.java

```diff
@@ -1,25 +1,34 @@
 package inheritance;
 
+
+import java.util.ArrayList;
+
 public class Restaurant {
-    private final String name;
-    private final int starRating;
-    private final int numOfDollarSigns;
+    public String name;
+    public int starRating;
+    public String priceCategory;
+    public ArrayList<Review> reviews = new ArrayList<>();
 
 // Constructor
-    public Restaurant(String name, int starRating, int numOfDollarSigns){
-
+    public Restaurant(String name, int starRating, String priceCategory){
         this.name = name;
         this.starRating = starRating;
-        this.numOfDollarSigns = numOfDollarSigns;
+        this.priceCategory = priceCategory;
     }
 
 //  Add Review
-    public void addReview(){
-
+    public void addReview(Review review){
+//  Checks if review already exists in ArrayList
+        if (!reviews.contains(review)){
+//  Attaches review to restaurant & adds review to reviews ArrayList
+            this.reviews.add(review);
+//  Updates star rating
+            this.starRating = review.reviewRating;
```

Comment:
This will reset the starRating to be the most recently added review's star rating. This means that users won't get the restaurant's average star rating!